roles/StackStorm.st2web/defaults/main.yml

```diff
@@ -7,3 +7,6 @@ st2web_ssl_certificate: null
 
 # String with custom SSL certificate private key. If not provided, self-signed certificate will be generated.
 st2web_ssl_certificate_key: null
+
+# A string that represents an custom nginx config file
```

Comment:
The var description in README.md is pretty good :+1: 
Please include that var description here as well.